README.md

```diff
@@ -1,110 +1,89 @@
 # libres [![Libres testing](https://github.com/equinor/libres/workflows/Libres%20testing/badge.svg)](https://github.com/equinor/libres/actions?query=workflow%3A%22Libres+testing%22) [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
-`libres` is part of the `ERT` project: _[Ensemble based Reservoir Tool](https://github.com/Equinor/ert)_.
+*libres* is part of the `ERT` project: _[Ensemble based Reservoir Tool](https://github.com/Equinor/ert)_. It is now available in PyPI:
 
-## Building libres
+``` sh
+$ pip install equinor-libres
+```
 
-### 1. Build libecl
-Build and install [libecl](https://github.com/Equinor/libecl) at the version that `.libecl_version` specifies. When configuring
-`libecl` you should used the option `-DCMAKE_INSTALL_PREFIX` to tell ``cmake``
-where to install `libecl`. The value passed to `CMAKE_INSTALL_PREFIX` will be
-needed when running cmake to configure `libres` in point 4 below. For now let us
-assume that the prefix `/local/ert/install` was used.
-   
-   
-### 2. Install Python dependencies
+or, for the latest development version (requires GCC/clang and `Python.h`):
 
+``` sh
+$ pip install git+https://github.com/equinor/libres.git@master
 ```
-pip install -r requirements.txt 
-```
 
-### 3. Update environment variables 
-To ensure that the build system correctly finds the `ecl` Python package you
-need to set the environment variables `PYTHONPATH` and `LD_LIBRARY_PATH` to
-include the `libecl` installation:
-  
+## Development
+
+*libres* is meant to be installed using `setup.py`, directly or using `pip
+install ./`. The `CMakeLists.txt` exists, but is used by `setup.py` to generate
+the `libres` C library and by Github Actions to run C tests.
+
+### Building
+
+Use the following commands to start developing from a clean virtualenv
 ```
-bash% export LD_LIBRARY_PATH=/local/ert/install/lib64:$LD_LIBRARY_PATH
-bash% export PYTHONPATH=/local/ert/install/lib/python3.6/site-packages:$PYTHONPATH
+$ pip install -r requirements.txt
+$ python setup.py develop
 ```
 
-Observe that path components `lib64` and `lib/python3.6/site-packages` will
-depend on your Python version and which Linux distribution you are using. The
-example given here is for RedHat based distributions.
+Alternatively, `pip install -e .` will also setup `libres` for development, but
+it will be more difficult to recompile the C library.
```

Comment:
How do you recompile in the two different scenarios?

Reply:
I'll add this:

[scikit-build](https://scikit-build.readthedocs.io/en/latest/index.html) is used for compiling the C library. It creates a directory named `_skbuild` which is reused upon future invocations of either `python setup.py develop`, or `python setup.py build_ext` which only rebuilds the C library. In some cases this directory must be removed in order for compilation to succeed.